community/portworx/templates/portworx-ds.yaml

```diff
@@ -255,45 +217,10 @@ spec:
               mountPath: /var/run/dbus
             - name: hostproc
               mountPath: /host_proc              
-          {{- else if eq (.Values.deploymentType | upper | lower) "docker" }}
-            - name: dev
-              mountPath: /dev
-            - name: optpwx
-              mountPath: /export_bin
-            - name: dockerplugins
-              mountPath: /run/docker/plugins
-            - name: hostproc
-              mountPath: /hostproc
-          {{- if semverCompare "< 1.10-0" .Capabilities.KubeVersion.GitVersion }}
-            - name: libosd
-              mountPath: /var/lib/osd:shared
-          {{- if (.Values.openshiftInstall) and (eq .Values.openshiftInstall true)}}
-            - name: kubelet
-              mountPath: /var/lib/origin/openshift.local.volumes:shared
-          {{- else }}
-            - name: kubelet
-              mountPath: /var/lib/kubelet:shared
-          {{- end }}
-
-          {{- else }}
-            - name: libosd
-              mountPath: /var/lib/osd
-              mountPropagation: "Bidirectional"
-          {{- if (.Values.openshiftInstall) and (eq .Values.openshiftInstall true)}}
-            - name: kubelet
-              mountPath: /var/lib/origin/openshift.local.volumes
-              mountPropagation: "Bidirectional"
-          {{- else }}
-            - name: kubelet
-              mountPath: /var/lib/kubelet
-              mountPropagation: "Bidirectional"
-          {{- end }}
-
-          {{- end }}
-            - name: src
-              mountPath: /usr/src
-          {{- end }}
-
+            - name: criosock
```

Comment:
should we not check first that if we are running on openshift before adding these path(s)?

Reply:
Don't need to. If the path doesn't exist on the source it'll create it. Removes the cluttering of conditions that we had.